### PR TITLE
UI change using flex

### PR DESCRIPTION
**Pull request by rkishan516**
Fixes #15 
UI Change Using Flex

screenshot from 2018-12-31 15-13-00

### CODE REVIEW

**Comment by rkishan516** [comment]
@akshatnitd Please check this out.

**Review by akshatnitd — COMMENTED** [review]
Make the number of links in each row consistent. Keep it 3 per row.

**Comment by rkishan516** [comment]
screenshot from 2019-01-03 08-22-19
@akshatnitd Please look at this. Is it okay.

**Comment by rkishan516** [comment]
@akshatnitd Please look at this ASAP .

**Comment by rkishan516** [comment]
@akshatnitd Only 2 Days left So,Please check these PR's ASAP.

**Comment by rkishan516** [comment]
@akshatnitd Only 2 Days left
So,Please check these PR's ASAP.

File: js/main.css [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
-	width: 167.5px;
 	clear: both;
 	cursor: pointer;
 	background-color:#424242;
 	padding-top: 3px;
 	padding-bottom: 3px;
+	flex: 1;
+}
+
+#item1
```
Comment (akshatnitd) [inline js/main.css]: Use same class for all the 3 items, instead of using 3 ids

**Review by akshatnitd — CHANGES_REQUESTED** [review]

File: js/main.css (hunk 1 above)
Comment (rkishan516) [reply js/main.css]: but using same class we can't add 3 column's

Comment (rkishan516) [reply js/main.css]: @akshatnitd Please response

Comment (rkishan516) [reply js/main.css]: @akshatnitd please rest asap.

Comment (rkishan516) [reply js/main.css]: @akshatnitd please look at this asap.

Comment (akshatnitd) [reply js/main.css]: Classes to each item isn't required.
Use - 
```
display: flex;
flex-direction: row;
flex-wrap: wrap;
```

Comment (rkishan516) [reply js/main.css]: @akshatnitd That give the previous shown result not in group of 3.

File: popup.js [hunk 2]
```diff
@@ -53,19 +52,42 @@ var current_tab_data = () => {
...
-            }else if(second_tab == 0){
-                str3 = '<tr><td><div class="list-group-item" id="item">'+'<div class="circle"><img src='+f+' /></div> <div id="text"><center>'+t+'</center></div></div></td>';
-                second_tab++;
-            }else{
-                str3 = '<td><div class="list-group-item" id="item">'+'<div class="circle"><img src='+f+' /></div> <div id="text"><center>'+t+'</center></div></div></td>';
-                second_tab++;
+            str3= '<div class="list-group-item" id="item">'+'<center><div class="circle"><img src='+f+' /></div> <div id="text">'+t+'</div></center></div>';
+            str2+=str3;
+        }
+        str2 += '</div><div id = "#item0">';
```
Comment (akshatnitd) [inline popup.js]: Do not use `id` here. Use class and then add css to that class.

File: popup.js [hunk 3]
```diff
@@ -53,19 +52,42 @@ var current_tab_data = () => {
...
+        for(j=2;j<no_of_tabs;j += 3) {
+            tab=tabs_in_window[j];
+            var f = tab.favIconUrl, t = tab.title, u = tab.url;
+            if (!u) u = '';
+            if (!t) t = u;
+            if(t.length > 17){
```
Comment (akshatnitd) [inline popup.js]: Incorrect approach. Use `text-overflow: ellipsis` css property.

File: popup.js [hunk 4]
```diff
@@ -104,19 +125,42 @@ var saved_tab_data =  () => {
...
+            str5+=str6;
+        }
+        str5 += '</div><div id = "#item0">';
+        for(m=1;m<no_of_tabs;m += 3) {
+            tab=tabs_in_window[m];
+            var f = tab.favIconUrl, t = tab.title, u = tab.url;
+            if (!u) u = '';
+            if (!t) t = u;
+            if(t.length > 17){
+                t = t.substring(0,18);
```
Comment (akshatnitd) [inline popup.js]: Same here

**Review by akshatnitd — CHANGES_REQUESTED** [review]

**Comment by rkishan516** [comment]
screenshot from 2019-01-06 12-32-08
@akshatnitd Now Extension is look like this.

**Comment by rkishan516** [comment]
@akshatnitd I am travelling to college , So may maximumly contribute for next half an hour, So Please Reply ASAP.

**Comment by rkishan516** [comment]
@akshatnitd Please review this ASAP.

**Review by akshatnitd — COMMENTED** [review]
The elipsis functionality is not working. Please check. Ask to review, once it is ready to merge.

**Comment by rkishan516** [comment]
screenshot from 2019-01-06 20-01-55
Now Extension is looking like this.
@akshatnitd All 3 PR's are Ready to Review.

**Review by akshatnitd — APPROVED** [review]
It looks good now! Thanks for your contribution.

**Pull request merged**